compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -1103,6 +1103,66 @@ tests Tools{damlc,repl,validate,davlDar,oldProjDar} = testGroup "Data Dependenci
             ]
         withCurrentDirectory (tmpDir </> "proj") $
             callProcessSilent damlc ["build"]
+
```

Comment:
sort of related: we probably have to take another stab at optimizing those tests. I’ve seen a few timeouts recently.